model/job/scheduler.go

```diff
@@ -60,6 +60,7 @@ type (
 	TriggerInfos struct {
 		TID          string                 `json:"_id,omitempty"`
 		TRev         string                 `json:"_rev,omitempty"`
+		Cluster      int                    `json:"cluster,omitempty"`
```

Comment:
Probably this should be named `CouchCluster` or `DBCluster` to avoid future collision.